core/vm/instructions.go

```diff
@@ -518,7 +518,8 @@ func opSload(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]by
 }
 
 func opSstore(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte, error) {
-	if interpreter.readOnly {
+	// if scope is an account with no code (EOA delegation through EIP-5806) sstore is protected
+	if interpreter.readOnly || scope.Contract.CodeHash == (common.Hash{}) {
```

Comment:
@gballet does that look ok ?

Reply:
Just realized, this may block sstore that happens during create (before the code/codehash is set).

Reply:
Actually, in `core/vm/evm.go:create(...)`, the contract code and hash is set on line 457, before the interpreteris called on line 467. So this might be good.

Reply:
nope, but I'll fix it in a minute.